### PR TITLE
fix: resolve broken arrow top position(fix #593)

### LINKED ISSUE #593: Incorrect placement the arrow in the schedule details popup
## To Reproduce
1. In the weekly view (`default: 'week'`)
2. Make a long schedule.(e.g. 04/27 00:30 ~ 23:00)
3. The category of schedule is 'time'. (`category: 'time'`)
4. Click this schedule.
5. Scroll down and check the placement of the detail popup.
스크린샷 2020-04-27 20 26 29

## Expected behavior
Do not overflow the container.

스크린샷 2020-04-27 20 35 00

### PR DESCRIPTION
**Pull request by jungeun-cho**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
fix: resolve broken arrow top position(fix #593)

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/js/view/popup/scheduleDetailPopup.js [hunk 1]
```diff
@@ -191,6 +191,10 @@ ScheduleDetailPopup.prototype._getYAndArrowTop = function(
         y -= containerTop;
     }
 
+    if (arrowTop < 0 || arrowTop > layerHeight) {
+        arrowTop = '50%';
```
Comment (seonim-ryu) [inline src/js/view/popup/scheduleDetailPopup.js:195]: `arrowTop` 타입이 `number`이고 top 값에 해당되면 사용하는 곳에서 `px` 단위로 변환을 할 것 같은데요. `%` 값을 사용해도 문제가 없나요??

Comment (dongsik-yoo) [reply src/js/view/popup/scheduleDetailPopup.js:195]: `'px'`로 쓰고 있네요.
https://github.com/nhn/tui.calendar/blob/f11b7a8b331bbc53570993307ef4a70b27800491/src/js/view/popup/scheduleDetailPopup.js#L307-L309

Comment (jungeun-cho) [reply src/js/view/popup/scheduleDetailPopup.js:195]: `undefiend`값을 할당해주어야합니다.

Comment (jungeun-cho) [reply src/js/view/popup/scheduleDetailPopup.js:195]: `null`로 할당합니다.

**Pull request merged**